.github/workflows/server-tests.yml

```
@@ -70,6 +70,6 @@ jobs:
         run: bash ${GITHUB_WORKSPACE}/.github/helper/install.sh
 
       - name: Run Tests
-        run: cd ~/frappe-bench/ && bench --site test_site run-tests --app erpnext
+        run: cd ~/frappe-bench/ && bench --site test_site run-tests --app erpnext --coverage
```

```suggestion
        run: cd ~/frappe-bench/ && bench --site test_site run-tests --app erpnext
```
I'll add coverage command separately.